src/lang/terms/triple.pl

```diff
@@ -1,5 +1,8 @@
 :- module(lang_triple,
-		[ mng_triple_doc(t,-,t) ]).
+		[ 
+			mng_triple_doc(t,-,t),
+			triple(r,r,t) 
```

Comment:
I think every argument must be *t* e.g. you can write `triple(s, in([foo:A,baz:B], o)`